src/App.js

```diff
@@ -1,9 +1,11 @@
-
+import Display from './components/Display';
+import ButtonPanel from './components/ButtonPanel';
 
 function App() {
   return (
     <div className="App">
```

Comment:
**REQUIRED**
Has children components wrapped in [React.Fragment](https://reactjs.org/docs/fragments.html#short-syntax) short syntax ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone2/app/ReactFragment.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/44896092/))